packages/next/migrations.json

```diff
@@ -23,6 +23,12 @@
       "version": "16.4.0-beta.3",
       "description": "Update package.json moving @nx/next from dependency to devDependency",
       "implementation": "./src/migrations/update-16-4-0/update-nx-next-dependency"
+    },
+    "update-17-2-7": {
+      "cli": "nx",
+      "version": "17-2-7",
```

Comment:
I'm not sure if this syntax (`17-2-7`) has the same effect as the dot syntax (`17.2.7`), but at least for the sake of consistency, should we use `17.2.7` instead?

Reply:
Yeah, it doesn't. It was a typo.